image/templates/common/setup-scanner-v4.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Sets up Central to use Scanner-v4. Note that deploying Scanner is a prerequisite to deploying Scanner-v4.
+
+KUBE_COMMAND=${KUBE_COMMAND:-{{.K8sConfig.Command}}}
+NAMESPACE="${ROX_NAMESPACE:-stackrox}"
+
+${KUBE_COMMAND} -n "$NAMESPACE" patch deploy/central -p '{"spec":{"template":{"spec":{"containers":[{"name":"central","env":[{"name": "ROX_SCANNER_V4", "value": "true"}]}]}}}}'
```

Comment:
Looks good technically, but I find such overlong lines tricky to read (and check). What do you think about something like this (taken from `run-scanner-v4.bats`):

```
"${KUBE_COMMAND}" -n ... patch ... --patch-file <(cat <<EOF
spec:
  template:
    spec:
      containers:
        - name: central
          env:
            - name: ROX_SCANNER_V4
              value: "true"
EOF
)
```